components/HomeHeader.vue

```diff
@@ -1,16 +1,19 @@
 <template>
   <swiper
-    v-if="items.length > 0 && !$vuetify.breakpoint.mobile"
+    v-if="items.length > 0 && !$vuetify.breakpoint.mobile && loaded"
```

Comment:
```suggestion
    v-if="items.length > 0 && !$vuetify.breakpoint.mobile && !loading"
```

We've been using `loading` everywhere else, it's probably best to keep this consistent.